### PR TITLE
fix(MessageView): Changed visual borders of message

### PR DESCRIPTION
**Pull request by igor-sirotin**
Fixes https://github.com/status-im/status-desktop/issues/7815

### What does the PR do

Moved `DateGroupLabel` out of `StatusMessage`. 
Now hovering works correctly.

### Affected areas

chat

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

Снимок экрана 2022-10-07 в 16 49 09

### CODE REVIEW

File: ui/imports/shared/views/chat/MessageView.qml [hunk 1]
```diff
@@ -433,11 +460,8 @@ Loader {
             hasExpired: root.isExpired
             reactionsModel: root.reactionsModel
 
-            previousMessageIndex: root.prevMessageIndex
-            previousMessageTimestamp: root.prevMsgTimestamp
-
             showHeader: root.authorCurrentMsg !== root.authorPrevMsg ||
-                        root.shouldRepeatHeader || dateGroupVisible || isAReply
+                        root.shouldRepeatHeader || dateGroupLabel.visible || isAReply
```
Comment (caybro) [inline ui/imports/shared/views/chat/MessageView.qml:464]: `dateGroupLabel.visible`: This is inside a `Component`!

**Review by caybro — CHANGES_REQUESTED** [review]
Otherwise looks good

File: ui/imports/shared/views/chat/MessageView.qml (hunk 1 above)
Comment (caybro) [reply ui/imports/shared/views/chat/MessageView.qml]: Ouch, this is impossible to review now :/

**Review by caybro — APPROVED** [review]

**Comment by iurimatias** [comment]
@igor-sirotin PR needs rebase

**Review by Khushboo-dev-cpp — APPROVED** [review]
LGTM

**Review by elina2015 — APPROVED** [review]
Tested and approved

**Pull request merged**